frontend/main-app/src/index.tsx

```diff
@@ -1,9 +1,6 @@
 import React from "react";
 import { createRoot } from "react-dom/client";
-
-const App = (): React.JSX.Element => (
-  <div>If you see this, then form-viewer has successfully loaded! 🚀</div>
-);
+import { App } from "./app/App";
```

Comment:
```suggestion
import { App } from "./App";
```

Reply:
Good thing that you pointed out the App import, otherwise I would have missed it 😄